TESTING.md

```diff
@@ -0,0 +1,162 @@
+# Testing Guide
+
+This document describes how to:
+- create and run tests for your development work,
+- ensure code meets PHP and WordPress Coding Standards, for best practices and security,
+- ensure code passes static analysis, to catch potential errors that tests might miss
+
+If you're new to creating and running tests, this guide will walk you through how to do this.
+
+For those more experienced with creating and running tests, our tests are written in PHP using [Playwright](https://playwright.dev/)
+and [PHPUnit](https://phpunit.de/).
+
+A PHPUnit guide for WordPress can be found [here](https://make.wordpress.org/core/handbook/testing/automated-testing/phpunit/).
+
+## Prerequisites
+
+If you haven't yet set up your local development environment with this Plugin repository installed, refer to the [Setup Guide](SETUP.md).
+
+If you haven't yet created a branch and made any code changes to the Plugin or Theme, refer to the [Development Guide](DEVELOPMENT.md)
+
+## Write (or modify) a test
```

Comment:
I would add a link to https://github.com/Codeinwp/neve/blob/master/e2e-tests/README.md as the process is more detailed there